### DISCUSSION TITLE
Search callsign field delete/clear button

### DISCUSSION THREAD

**Discussion by dimarmnes** [foldynl/QLog#395] [Ideas]
Hi,
Several times I felt the lack of a button to clear the callsign search field like in the CW Console.

![qlog_del_callsign_search](https://github.com/foldynl/QLog/assets/168468132/d74c6034-1bd5-4ab8-b028-8ada666acac9)

**Comment by foldynl** [comment]
that's a good point. yes, that makes sense. Opening an issue for it

**Comment by foldynl** [comment]
It is a part of v0.37.0
